src/measure_theory/measure/finite_measure_weak_convergence.lean

```diff
@@ -154,6 +158,213 @@ def coe_add_monoid_hom : finite_measure α →+ measure α :=
 instance {α : Type*} [measurable_space α] : module ℝ≥0 (finite_measure α) :=
 function.injective.module _ coe_add_monoid_hom finite_measure.coe_injective coe_smul
 
+/-- The pairing of a finite (Borel) measure `μ` with a nonnegative bounded continuous
+function is obtained by (Lebesgue) integrating the (test) function against the measure.
+This is `finite_measure.test_against'`. -/
+abbreviation test_against_nn (μ : finite_measure α) (f : α → nnreal) : ℝ≥0 :=
+(∫⁻ x, f x ∂(μ : measure α)).to_nnreal
+
+variables [topological_space α]
+
+-- I believe the formulation is generally useful, except maybe the exact form
+-- of the assumption `f_bdd`.
+-- Where to place?
+lemma _root_.is_finite_measure.lintegral_lt_top_of_bounded_to_ennreal {α : Type*}
+  [measurable_space α] (μ : measure α) [μ_fin : is_finite_measure μ]
+  {f : α → ℝ≥0∞} (f_bdd : ∃ c : ℝ≥0, ∀ x, f x ≤ c) : ∫⁻ x, f x ∂μ < ∞ :=
```

Comment:
Could you use `bdd_above` in which case you can prove the lemma using `set_lintegral_lt_top_of_bdd_above` and `set_lintegral_univ` (you might need to pull from master).

Reply:
Thanks for the comment!

This sounded good, and I actually already started changing. But then I noticed that `set_lintegral_lt_top_of_bdd_above` has a measurability assumption on the function `f`, and I recalled that the reason I had separated this to a standalone lemma with more general assumptions than my use case was a suggestion by @hrmacbeth.

Both are doable, but my current feeling is that it is also useful to have this as a lemma under the typeclass assumption `is_finite_measure` (rather than the explicitly provided finiteness of the measure of the set in question) and without measurability assumption and not a `set_integral_xxx`. There would be a bit of code duplication, but in my opinion not too much, and the two both versions should have their legitimate use cases.

So after all, my preference would be to keep the original version. Thank you for the suggestion in any case!

Reply:
One more thing: when the function is taken `ennreal`-valued as originally here, `bdd_above` does not work (since `∞` is a stupid upper bound). If I remember correctly, at some point Floris suggested a notion of "properly upper boundedness" for types with `has_top`, which I think might be a generally good idea, but with this PR there is still so little use for it that I would postpone that consideration.

Reply:
I see, we can keep it as it is then :)